Source/Applications/SystemCenter/Controllers/ExternalDB/LineSegmentWizardController.cs

```diff
@@ -207,24 +207,25 @@ public IHttpActionResult PostData(int id, [FromBody] LineConfiguration record)
                 }
             }
 
-            Func<string> generatekey = () =>
+            Func<(string key, string segmentNumber)> generatekey = () =>
```

Comment:
```suggestion
            Func<string> generatekey = () =>
```

return the segment number as a string only and turn it into a Key and Name to make this a little easier to read. Also matches TSX if you do that